mapstory/api/resourcebase_api.py

```diff
@@ -211,32 +210,39 @@ def build_haystack_filters(self, parameters):
                     SQ(content__exact=phrase)
                 )
             else:
+                print 'WE FOUND IT'
```

Comment:
May want to remove the debug statements or use a logger.

Reply:
Whoops, fixed.